packages/gatsby-theme-carbon/src/components/ImageCard/image-card.scss

```diff
@@ -3,11 +3,34 @@
   padding: $spacing-05 25% $spacing-05 $spacing-05;
   position: relative;
   text-decoration: none;
+
+  &::after {
+    position: absolute;
+    top: rem(3px);
+    left: rem(3px);
+    content: '';
+    width: calc(100% - 7px);
+    height: calc(100% - 7px);
+    background: none;
+    outline: 3px solid transparent;
+    transition: outline motion(standard, productive) $duration--fast-01;
```

Comment:
rather than the 3px `top` and `left` we can just use `outline-offset`, and rather than the `100% - 7px` for `width` and `height` calculation we can just reduce the dimensions by 1px (because of https://github.com/carbon-design-system/gatsby-theme-carbon/blob/master/packages/gatsby-theme-carbon/src/components/ImageCard/image-card.scss#L62-L63). I know the width and height offset value is 7 to account for 2 sides * 3px + 1px from the gap but that might not be obvious at first glance, so I figured keeping it at 1px in reference to L62-63 might be clearer

```scss
&::after {
  position: absolute;
  top: 0;
  left: 0;
  content: '';
  width: calc(100% - 1px); // 1px to offset negative `left` positioning
  height: calc(100% - 1px); // 1px to offset negative `top` positioning
  outline: 3px solid transparent;
  outline-offset: -3px;
  transition: outline motion(standard, productive) $duration--fast-01;
}
```